publicodes/source/mecanisms/unité.tsx

```diff
@@ -0,0 +1,60 @@
+import { typeWarning } from '../error'
+import { makeJsx, registerEvaluationFunction } from '../evaluation'
+import { evaluationFunction } from '..'
+import {
+	areUnitConvertible,
+	convertUnit,
+	parseUnit,
+	serializeUnit
+} from '../units'
+
+function MecanismUnité({ explanation, nodeValue, unit }) {
+	return makeJsx({ ...explanation, nodeValue, unit })
+}
+
+const evaluate: evaluationFunction = node => {
+	const valeur = this.evaluate(node.explanation)
```

Comment:
Plutôt `this.evaluateNode`, il faudrait fusionner ces deux fonctions (et utiliser `this.evaluate` partout), mais le point d'entrée public fait quelques opérations supplémentaires cf. publicodes/index.

Par ailleurs on est obligé d'utilisé une `function(node)` et non une `=>` pour pouvoir utiliser `this`.

Reply:
J'ai vu ça :+1: